konfera/static/konfera/css/pycon.css

```diff
@@ -0,0 +1,656 @@
+  table.schedule .speaker {
+    text-align: right !important;
+  }
+  .col-md-6.narrow {
+    padding-right: 4px;
+    padding-left: 4px;
+  }
+}
+@media (max-width: 400px) {
+  table.schedule .speaker {
+    text-align: right !important;
+  }
+}
+
+#event-details .modal-footer h3 {
+  font-size: 18px;
+  margin-top: 2px;
+  margin-bottom: 4px;
+}
+
+#event-details .modal-footer p {
+  font-size: 12px;
+}
+
+#speaker-bio .img-thumbnail, #speaker2-bio .img-thumbnail {
+  margin-top: 10px;
+  max-height: 120px;
+}
+
+/*
+A custom Bootstrap 3.2 'Google Plus style' theme
+from http://bootply.com
+
+This CSS code should follow the 'bootstrap.css'
+in your HTML file.
+
+license: MIT
+author: bootply.com
+*/
+
+/*
+@import url(http://fonts.googleapis.com/css?family=Roboto:400);
+body {
+  background-color:#e0e0e0;
+  -webkit-font-smoothing: antialiased;
+  font: normal 14px Roboto,arial,sans-serif;
+}
+*/
+
+.navbar-default {background-color:rgba(244, 244, 244, 0.9);border-width:0;z-index:5;}
+.navbar-default .navbar-nav > li > a {border-bottom: 2px solid transparent;}
+.navbar-default .navbar-nav > .active > a,.navbar-default .navbar-nav > li:hover > a {border:0 solid #4285f4;border-bottom-width:2px;background-color:transparent;color:#000;text-shadow:0px 1px 1px rgba(0, 0, 0, 0.4)}
+.navbar-default .navbar-nav.lang-nav > .active > a,.navbar-default .navbar-nav.lang-nav > li:hover > a {font-weight: normal;}
+.navbar-default .navbar-header.active > a .navbar-brand, .navbar-default .navbar-header:hover > a .navbar-brand {color:#000; text-shadow:0px 1px 1px rgba(0, 0, 0, 0.2)}
+.navbar-default .dropdown-menu {background-color:rgba(244, 244, 244, 0.9); z-index:5;}
+.navbar-default .dropdown-menu li > a {padding-left:20px;}
+.navbar-default .dropdown-menu li.active > a, .navbar-default .dropdown-menu li > a:hover {
+  background-color:rgba(66, 133, 244, 0.9);
+}
+
+.header {background-color:#ffffff;border-width:0;}
+.header .navbar-collapse {background-color:#ffffff;}
+.btn,.form-control,.panel,.list-group,.well {border-radius:1px;box-shadow:0 0 0;}
+.form-control {border-color:#d7d7d7;}
+.btn-primary {border-color:transparent;}
+.btn-primary,.label-primary,.list-group-item.active, .list-group-item.active:hover, .list-group-item.active:focus {background-color:#4285f4;}
+.btn-plus {background-color:#ffffff;border-width:1px;border-color:#dddddd;box-shadow:1px 1px 0 #999999;border-radius:3px;color:#666666;text-shadow:0 0 1px #bbbbbb;}
+.well,.panel {border-color:#d2d2d2;box-shadow:0 1px 0 #cfcfcf;border-radius:3px;}
+.btn-success,.label-success,.progress-bar-success{background-color:#65b045;}
+.btn-info,.label-info,.progress-bar-info{background-color:#a0c3ff;border-color:#a0c3ff;}
+.btn-danger,.label-danger,.progress-bar-danger{background-color:#dd4b39;}
+.btn-warning,.label-warning,.progress-bar-warning{background-color:#f4b400;color:#444444;}
+
+hr {
+  border-color:#ececec;
+}
+input, select {
+  width: 98%;
+  margin: 1px 0;
+}
+button {
+ outline: 0;
```

Comment:
In most of the file we use 2 spaces, here is just one (same applies for some other lines). Let's keep it consistent.